pkg/apis/monitoring/v1alpha1/scrapeconfig_types.go

```diff
@@ -390,3 +398,51 @@ type EC2SDConfig struct {
 	// +optional
 	Filters []*EC2Filter `json:"filters"`
 }
+
+type ProxyConfig struct {
+	// HTTP proxy server to use to connect to the targets.
```

Comment:
(suggestion) the proxy config might apply to the scrape HTTP client or to the service discovery HTTP client so it's better to be generic here.
```suggestion
	// `proxyURL` defines the HTTP proxy server to use.
```